eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -54,9 +54,16 @@ jobs:
           versionSpec: "3.6"
       - script: |
           pip install setuptools wheel
-          pip install doc-warden
-          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
-        displayName: "Verify Readmes"
+          pip install doc-warden==0.4.2
+          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml -t all
+        displayName: "Verify Readmes and changelogs"
```

Comment:
Lets not merge this until we get the version incrementing scripts updating the change logs appropriately.